src/shop/__init__.py

```diff
@@ -0,0 +1,35 @@
+import typing
+
+from src.shop.customer import Customer
+from src.shop.order import Order
+from src.shop.item import Item
+from src.shop.product import *
+
+
+def notify_everyone(orders: typing.List[Order]):
+    for order in orders:
+        order.customer.notify()
+
+def place_order(customer, *items):
+    '''
+    # customer jako obiekt klasy, items jako lista [produkt, ilość"
+    - nie umiem tego zapisać poprzez typowanie
+    '''
+    order = Order(customer)
+    for i in items:
+        order.add_item(Item(i[0], i[1]))
```

Comment:
Jeżeli chodzi o sam typ wejściowy, to bardzo dobrze byłoby przekazywać już zbudowany `Item`. Aczkolwiek nie jest błędem budowanie go wewnątrz tej funkcji. Wszystko zależy od tego jakie przyjmiemy założenia 😉 

Natomiast to co jest warte zauważenia, to fakt, że zmienna `i` wygląda zawsze tak samo.
Co sądzisz o użyciu funkcjonalności rozpakowywania krotki/listy zamiast dobierać się do interesujących nas elementów poprzez indeksy?